### DISCUSSION TITLE
Not in Database FMC Message

### DISCUSSION THREAD

**Discussion by emsss78** [flybywiresim/aircraft#7116] [Help]
Hi all,

I keep getting this message after loading the Simbrief flight plan even though my Navigraph NavData Center app is updated to the latest airac cycle (now AIRAC Cycle 2204 rev.1) any thoughts on that?? This happens in all the FBW versions. Oh by the way my Navigraph Subscription is also unlocked and active on the Simbrief website.

Thanks...

**Comment by tracernz** [comment]
Try removing and re-installing the data with the navdata center, and after that restarting your sim. If After that you still have issues try entering the flightplan by hand and see at which point you get an issue.

**Comment by emsss78** [comment]
Ok Thanks for the feedback @tracernz  I'll try the manual route input and see which waypoint is causing the issue as I already uninstalled and re-installed the data in navdata center and the message still remained.
